docs/release.md

```diff
@@ -57,6 +57,14 @@ Release notes
 -------------
 Every release must should be accompanied by release notes describing the
 major highlights of the release.
+
+If a commit message contains the following snippet, then the provided text will
+appear as a release note.
+
+   ```release-note
```

Comment:
Hm, my concern is, that you would still have to go over everything, since this is not enforced anywhere, and it is pretty hidden. From that perspective, it might be still more useful to go over all PRs which were merged in the meantime in the Github UI. The amount of PRs right now should be manageable for the next one or two releases. In that timeframe it should be possible to come up with a more visible approach (e.g. labels on github PRs).

Reply:
They are two distinct problems: One is to solve how the release notes are added the other is how to enforce this.

In general I'd really like to see that we have an automated way to generate release notes.
And obviously it would also be nice to then be able to enforce it.

Side note: Without the automation we still have the author of a change and the reviewers whoc can manually enforce this snippet (or in future set a label) if they think that a change should be noted.

Reply:
Sure two problems, but definitely connected in this case. This feature is so much hidden, that without automation making it visible, I would really have trouble accepting this. I will be the first one who forgets adding this, which means that if I would do a release, I would go through all PRs anyway. When I then go through the PRs, it will be hard for me to see if a PR is already covered by a release note.

So while I might be convinced to take this embedded release-note approach for collecting them, I would really be against starting that without any kind of notification. E.g. by automatically adding a release-note-none label, to notify all involved people that nothing will be collected from that PR. I also saw automation as a must-have when we agreed on coding styles and when we added code-generation.

Finally, what if a newer PR makes older release notes obsolete? What if they are wrong, but got merged? While that problem does not go away with another way of providing release notes, it would at least be much easier to somehow signal people on a PR that you don't want to have that included anymore, or to ask people, if it should still be included.

Reply:
Alright, dropped that part from the PR to move the remaining work forward.

Reply:
Alright, dropped that part from the PR to move the remaining work forward.